configure.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
```

Comment:
Here's a good strategy you may like to follow.

I see some hardcoded `export` calls in here.  I would propose the following.
1. Create a bash function to add this to your `~/.bashrc`.  You can assume the user here is using `bash` and not another shell such as `zsh` (the OSX default)
2. Add in the appropriate commands to build the USD python bindings.  (What steps you took to create the USD_python_build folder)

You can structure your bash script like this
```bash
#!/bin/bash

PY_BUILD_FOLDER=USD_python_build

build_python_bindings() {
  # bash steps to build the python bindings
  # should probably start with
  mkdir -p $PY_BUILD_FOLDER
  
  # and a handy command to `cd` into a folder to do things inside there is:
  pushd $PY_BUILD_FOLDER
  # do things
  popd
  # consider installing the USD python build to /usr/local/ (bin/lib)
  sudo rsync -a $PY_BUILD_FOLDER/ /usr/local
}

setup_paths() {
  export PYTHONPATH=$PYTHONPATH:/home/t_dhruh/Documents/Project/USD_python_build/lib/python
  export PATH=$PATH:/home/t_dhruh/Documents/Project/USD_python_build/bin
}

main() {
  build_python_bindings
  setup_paths
}
main

```